libs/core/build.gradle

```diff
@@ -21,38 +21,28 @@ apply plugin: 'nebula.optional-base'
 apply plugin: 'nebula.maven-base-publish'
 apply plugin: 'nebula.maven-scm'
 
-archivesBaseName = 'elasticsearch-core'
```

Comment:
Is there no way to change the module name but keep the project path the same? Having to write out the elasticsearch prefix when depending on these is extra boilerplate that is confusing, which is why we attempted to keep that prefix out of the internal name.

Reply:
> Is there no way to change the module name but keep the project path the same?

Not for the purposes of internal dependency resolution. I think this is one of those times where I gladly trade a bit of verbosity for consistency. I think having a consistent module name for both internal dependencies and external ones makes it a bit easier to reason about stuff. When I was digging though this stuff it was not immediately obvious that `elasticsearch-core == :libs:core`.